setup.cfg

```
@@ -50,7 +50,7 @@ include_package_data = True
 install_requires =
   attrs >= 17.3.0
   charset-normalizer >=2.0, < 3.0
-  multidict >=4.5, < 6.0
+  multidict >=4.5, < 7.0
```

Based on https://github.com/aio-libs/aiohttp/pull/6550#issuecomment-1020960474, I'd also exclude the problematic versions, I suppose:
```suggestion
  multidict >=4.5, < 7.0, != 6.0.0, != 6.0.1
```

---

_(decided not to add the exclusions)_